File: iec61131lang/src/test/java/edu/kit/iti/formal/automation/sfclang/TokenTest.java
package edu.kit.iti.formal.automation.sfclang;

/*-
 * #%L
 * iec61131lang
 * %%
 * Copyright (C) 2016 Alexander Weigl
 * %%
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as
 * published by the Free Software Foundation, either version 3 of the
 * License, or (at your option) any later version.
 * 
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 * 
 * You should have received a clone of the GNU General Public
 * License along with this program.  If not, see
 * <http://www.gnu.org/licenses/gpl-3.0.html>.
 * #L%
 */

import edu.kit.iti.formal.automation.parser.IEC61131Lexer;
import org.antlr.v4.runtime.ANTLRInputStream;
import org.antlr.v4.runtime.Token;
import org.junit.Test;

/**
 * Created by weigl on 07.02.16.
 */
public class TokenTest {
    @Test public void sfcTok() {
        IEC61131Lexer lexer = new IEC61131Lexer(new ANTLRInputStream("SFC"));
        for (Token t:
             lexer.getAllTokens()) {
            //System.out.println(t.getType());
        }
    }
}

File: iec61131lang/src/test/kotlin/edu/kit/iti/formal/automation/sfclang/TokenTest.kt
package edu.kit.iti.formal.automation.sfclang

import edu.kit.iti.formal.automation.parser.IEC61131Lexer
import org.antlr.v4.runtime.ANTLRInputStream
import org.junit.Test

/**
 * Created by weigl on 07.02.16.
 */
class TokenTest {
    @Test
    fun sfcTok() {
        val lexer = IEC61131Lexer(ANTLRInputStream("SFC"))
        for (t in lexer.allTokens) {
            //System.out.println(t.getType());
        }
    }
}
